decidim-core/app/controllers/decidim/editor_images_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
 
       CreateEditorImage.call(@form) do
         on(:ok) do |image|
-          render json: { url: image.attached_uploader(:file).path, message: I18n.t("success", scope: "decidim.editor_images.create") }
+          render json: { url: image.attached_uploader(:file).url(host: current_organization.host), message: I18n.t("success", scope: "decidim.editor_images.create") }
```

Comment:
I don't like this approach because this will be persisted to the database.

If the organization host is changed at some point, it would potentially break all the images within the instance embedded anywhere.